### PR TITLE
chore: clean up test createManager utility

### PR DESCRIPTION
**Pull request by EvanHahn**
_This is a test-only change. It is not urgent._

This fixes two small issues with the createManager test utility:

- Removes a `@ts-ignore` directive
- Avoid an unnecessary `t.teardown` call

These changes should have no impact.

(For some reason, I have wanted to tidy this every time I've looked at it...this
is me being nitpicky to the max.)

### CODE REVIEW

File: test-e2e/utils.js [hunk 1]
```diff
@@ -194,21 +194,28 @@ export async function createManagers(count, t, deviceType) {
...
+
+  if (FAST_TESTS) {
+    dbFolder = ':memory:'
+    coreStorage = () => new RAM()
+  } else {
+    const directories = [temporaryDirectory(), temporaryDirectory()]
+    ;[dbFolder, coreStorage] = directories
+    t.teardown(() =>
+      Promise.all(
+        directories.map((dir) =>
```
Comment (achou11) [inline test-e2e/utils.js:208]: countering your nitpick with my own nitpick 😄 (i always find the semi-colon trick(?) a little hard to parse)

```suggestion
    dbFolder = temporaryDirectory()
    coreStorage = temporaryDirectory()
    t.teardown(() =>
      Promise.all(
        [dbFolder, coreStorage].map((dir) =>
```

**Review by achou11 — APPROVED** [review]

File: test-e2e/utils.js (hunk 1 above)
Comment (EvanHahn) [reply test-e2e/utils.js:208]: This doesn't work because TypeScript doesn't know that `coreStorage` is a string at this point. We could fix that by defining an intermediate variable?

Comment (achou11) [reply test-e2e/utils.js:208]: ah welp nevermind 😂  - probably not worth going out of your way to change what you had before, especially since this is just test-specific
